Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -237,21 +224,23 @@ public static Result<IReadOnlyList<IBulkApplicationCommandData>> CreateApplicati
         // Perform validations
         if (commands.Count > _maxRootCommandsOrGroups)
         {
-            return new UnsupportedFeatureError
+            throw new UnsupportedFeatureException
             (
-                $"Too many root-level commands or groups (had {commands.Count}, max {_maxRootCommandsOrGroups})."
+                $"Too many root-level commands or groups (had {commands.Count}, max {_maxRootCommandsOrGroups}).",
```

Comment:
Nitpick: Isn't it more typical to have `Max: x, Found/Detected: y`?

Reply:
I agree, that would be more helpful.